go/pkg/rexec/rexec.go

```diff
@@ -363,7 +363,7 @@ func (ec *Context) DownloadOutErr() {
 // DownloadOutputs downloads the outputs of the command in the context to the specified directory.
 func (ec *Context) DownloadOutputs(execRoot string) {
 	st := ec.Result.Status
-	stats, res := ec.downloadOutputs(ec.cmd.ExecRoot)
+	stats, res := ec.downloadOutputs(execRoot)
```

Comment:
can we instead rename `execRoot` to `downloadDir` or something like that? IIUC this path has nothing to do with exec root - and it was clearly misleading :)

Reply:
Done.